content/v2.0-preview/get-started/get-started-with-managed-resources.md

```diff
@@ -188,44 +189,42 @@ spec:
 EOF
 ```
 
-The {{< hover label="xr" line="2">}}apiVersion{{< /hover >}} and 
-{{< hover label="xr" line="3">}}kind{{</hover >}} are from the provider's CRDs.
-
-
-The {{< hover label="xr" line="5">}}metadata.generateName{{< /hover >}} value is the 
-name of the created S3 bucket in AWS.  
-This example uses the generated name `crossplane-bucket-<hash>` in the 
-{{< hover label="xr" line="5">}}$bucket{{</hover >}} variable.
+The {{< hover label="xr" line="6">}}metadata.generateName{{< /hover >}} gives a
+pattern that the provider will use to create a unique name for the bucket in S3.
+The generated name will look like `crossplane-bucket-<hash>`.
```

Comment:
```suggestion
The {{< hover label="xr" line="6">}}metadata.generateName{{< /hover >}} gives a
pattern that Kubernetes will use to create a unique name for the bucket in S3.
The generated name will look like `crossplane-bucket-<hash>`.
```

Nit: this is more technically accurate. That said, I don't feel strongly if it feels less confusing to pretend it's the provider doing it in this context.